cscs-checks/libraries/gridtools/gridtools_base.py

```diff
@@ -0,0 +1,180 @@
+import reframe as rfm
+import reframe.utility.sanity as sn
+
+
+class GridToolsCheckBase(rfm.RegressionTest):
+    def __init__(self):
+        super().__init__()
+        self.descr = 'GridTools test base'
+
+        self.valid_prog_environs = ['PrgEnv-gnu']
+        self.prebuild_cmd = ['module switch gcc/6.2.0 gcc/5.3.0']# TODO: fix this
```

Comment:
Why do you do this here? Normally, the following should be enough:

```python
self.modules = ['gcc/5.3.0']
```

Also why do you need to switch back to such an old GCC version?

Reply:
There is a combination of supported gcc/nvcc versions in the GT wiki (https://github.com/GridTools/gridtools, "Supported compilers") and I took the oldest supported gcc version.